docs/topics/kubernetes-developers.md

```diff
@@ -0,0 +1,67 @@
+# For Kubernetes Developers
+
+If you are working on Kubernetes upstream, you can use Azure Cluster Api to test your build of Kubernetes in the Azure environment.
```

Comment:
What do you think about changing "Azure Cluster Api" to "the Cluster API Azure Provider"